<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotePad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>371</width>
      <height>186</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>NotePad</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actDoc_New"/>
   <addaction name="actDoc_Open"/>
   <addaction name="actDoc_Save"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionSave_All"/>
   <addaction name="actCloseALL"/>
   <addaction name="separator"/>
   <addaction name="actCut"/>
   <addaction name="actCopy"/>
   <addaction name="actPaste"/>
   <addaction name="separator"/>
   <addaction name="actionTabbedView"/>
   <addaction name="actCascade"/>
   <addaction name="actTile"/>
   <addaction name="separator"/>
   <addaction name="actionGo_to"/>
   <addaction name="actionsearch"/>
   <addaction name="actionReplace"/>
   <addaction name="actionSearch_Previous"/>
   <addaction name="actionSearch_Next"/>
   <addaction name="separator"/>
   <addaction name="actFont"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionzoomout"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actDoc_New"/>
    <addaction name="actDoc_Open"/>
    <addaction name="actionrecentfiles"/>
    <addaction name="separator"/>
    <addaction name="actDoc_Save"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actCloseALL"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actCopy"/>
    <addaction name="actPaste"/>
    <addaction name="actCut"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionGo_to"/>
    <addaction name="actionsearch"/>
    <addaction name="actionReplace"/>
    <addaction name="actFont"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionTabbedView"/>
    <addaction name="actCascade"/>
    <addaction name="actTile"/>
   </widget>
   <widget class="QMenu" name="menuVew">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionzoomout"/>
    <addaction name="actFont"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuVew"/>
   <addaction name="menuWindow"/>
  </widget>
  <action name="actDoc_New">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/100.bmp</normaloff>:/images/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文档窗口</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本系统</string>
   </property>
  </action>
  <action name="actDoc_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/122.bmp</normaloff>:/images/images/122.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actFont">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/506.bmp</normaloff>:/images/images/506.bmp</iconset>
   </property>
   <property name="text">
    <string>字体设置</string>
   </property>
   <property name="toolTip">
    <string>字体设置</string>
   </property>
  </action>
  <action name="actCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/200.bmp</normaloff>:/images/images/200.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/202.bmp</normaloff>:/images/images/202.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/204.bmp</normaloff>:/images/images/204.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionTabbedView">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/230.bmp</normaloff>:/images/images/230.bmp</iconset>
   </property>
   <property name="text">
    <string>Tab模式</string>
   </property>
   <property name="toolTip">
    <string>窗口模式或页面模式</string>
   </property>
  </action>
  <action name="actCascade">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/400.bmp</normaloff>:/images/images/400.bmp</iconset>
   </property>
   <property name="text">
    <string>级联展开</string>
   </property>
   <property name="toolTip">
    <string>窗口级联展开</string>
   </property>
  </action>
  <action name="actTile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/406.bmp</normaloff>:/images/images/406.bmp</iconset>
   </property>
   <property name="text">
    <string>平铺展开</string>
   </property>
   <property name="toolTip">
    <string>窗口平铺展开</string>
   </property>
  </action>
  <action name="actCloseALL">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/128.bmp</normaloff>:/images/images/128.bmp</iconset>
   </property>
   <property name="text">
    <string>关闭全部</string>
   </property>
   <property name="toolTip">
    <string>关闭所有窗口</string>
   </property>
  </action>
  <action name="actDoc_Save">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/104.bmp</normaloff>:/images/images/104.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存修改</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionGo_to">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/icons8-location-16.png</normaloff>:/images/images/icons8-location-16.png</iconset>
   </property>
   <property name="text">
    <string>Go to</string>
   </property>
   <property name="toolTip">
    <string>Go to(&amp;G)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/icons8-zoom-in-16.png</normaloff>:/images/images/icons8-zoom-in-16.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+=</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionzoomout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/icons8-zoom-out-16.png</normaloff>:/images/images/icons8-zoom-out-16.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+-</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionsearch">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/icons8-search-16.png</normaloff>:/images/images/icons8-search-16.png</iconset>
   </property>
   <property name="text">
    <string>Find(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>Show Find Dialog</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSearch_Next">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/icons8-next-48.png</normaloff>:/images/images/icons8-next-48.png</iconset>
   </property>
   <property name="text">
    <string>Find Next(F3)</string>
   </property>
   <property name="toolTip">
    <string>Find Next(F3)</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSearch_Previous">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/icons8-back-48.png</normaloff>:/images/images/icons8-back-48.png</iconset>
   </property>
   <property name="text">
    <string>Search Previous</string>
   </property>
   <property name="toolTip">
    <string>Find Previous(Shit + F3)</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/icons8-find-and-replace-24.png</normaloff>:/images/images/icons8-find-and-replace-24.png</iconset>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="toolTip">
    <string>replace text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/206.bmp</normaloff>:/images/images/206.bmp</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/208.bmp</normaloff>:/images/images/208.bmp</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionClose_other_files">
   <property name="text">
    <string>Close other files</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/save_as.png</normaloff>:/images/images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="icon">
    <iconset theme="保存所有文件" resource="res.qrc">
     <normaloff>:/images/images/saveall1.bmp</normaloff>:/images/images/saveall1.bmp</iconset>
   </property>
   <property name="text">
    <string>保存所有文件</string>
   </property>
   <property name="toolTip">
    <string>保存所有文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionrecentfiles">
   <property name="text">
    <string>最近文件</string>
   </property>
   <property name="toolTip">
    <string>打开最近文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
